pages/glossary/background-colors-of-text.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Background Colors Of Text Nodes
+key: background-colors-of-text
+---
+
+The colors of all pixels, excluding the [foreground colors](#foreground-colors-of-text) pixels, in the [bounding boxes around text](#bounding-box-around-text) around a [visible](#visible) character in a [text node](https://dom.spec.whatwg.org/#text).
```

Comment:
```suggestion
The colors of all pixels, excluding the [foreground colors](#foreground-colors-of-text) pixels, in the [bounding boxes](#bounding-box-around-text) around [visible](#visible) characters in a [text node](https://dom.spec.whatwg.org/#text).
```

Reply:
We've changed this to test individual characters.